src/magnet.js

```diff
@@ -304,6 +305,14 @@ class Magnet {
     }
   }
 
+  /**
+   * Register environment variables
+   * @param {Object} config 
+   */
+  registerProcessEvnVariables_(config) {
+    process.env.API_ONLY = config.magnet.apiOnly      
```

Comment:
Would that be appropriate to have a default value for `apiOnly` in this file? https://github.com/wedeploy/magnet/blob/master/src/config.js#L8

Reply:
Thanks @pragmaticivan for the review. Added default value falso to apiOnly config

Reply:
Thanks @pragmaticivan